### PR TITLE
Update animals table

### PR DESCRIPTION
**Pull request by anas599**
In this PR:

- Add a column species of type string to animals
- Add transaction to Update the animal's table by setting the species column to digimon for all animals with a name ending in 'mon'.
- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
- Add a transaction to multiply any negative with to make it a positive integer
- Add quires for counting the number of animals and max, min weight ...etc

Screenshot 2023-03-28 at 1 49 54 PM
Screenshot 2023-03-28 at 1 50 09 PM
Screenshot 2023-03-28 at 1 59 11 PM
Screenshot 2023-03-28 at 2 04 55 PM
Screenshot 2023-03-28 at 2 05 20 PM
Screenshot 2023-03-28 at 3 14 52 PM
Screenshot 2023-03-28 at 3 37 41 PM
Screenshot 2023-03-28 at 3 41 59 PM
Screenshot 2023-03-28 at 3 42 58 PM
Screenshot 2023-03-28 at 3 45 05 PM
Screenshot 2023-03-28 at 3 54 31 PM
Screenshot 2023-03-28 at 3 54 42 PM
Screenshot 2023-03-28 at 4 01 05 PM
Screenshot 2023-03-28 at 4 11 31 PM

### CODE REVIEW

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @anas599👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Status: Invalid Code Review Request



You have submitted a project, that has relevant sections of the README removed. Kindly note that all section of the README is required and for this project, the only optional sections are the FAQ and the live demo sections.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: data.sql [hunk 1]
```diff
@@ -8,4 +8,25 @@ INSERT INTO animals(id, name, escape_attempts, neutered, weight_kg, date_of_birt
...
 INSERT INTO animals(id, name,escape_attempts, neutered, weight_kg, date_of_birth)
-VALUES (1, 'Agumon', 0, TRUE, 10.23,TO_DATE('Feb 3rd, 2020', 'Mon DDth, YYYY'));
\ No newline at end of file
+VALUES (1, 'Agumon', 0, TRUE, 10.23,TO_DATE('Feb 3rd, 2020', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 5, 'Charmander', 0, FALSE, 11,TO_DATE('Feb 8th, 2020', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 6, 'Plantmon', 0, TRUE, 5.7,TO_DATE('Nov 15th, 2021', 'Mon DDth, YYYY'));
```
Comment (ShadyShawkat) [inline data.sql]: - [x] Please change the `escape_attempts` for this record to `2` instead of `0`.

File: data.sql [hunk 2]
```diff
@@ -8,4 +8,25 @@ INSERT INTO animals(id, name, escape_attempts, neutered, weight_kg, date_of_birt
...
+VALUES ('Animal', 8, 'Angemon', 1, TRUE, 45 ,TO_DATE('Jun 12th, 2005', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 9, 'Boarmon', 7, TRUE, 20.4 ,TO_DATE('Jun 7th, 2005', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 10, 'Blossom', 3, TRUE, 17 ,TO_DATE('Oct 13th, 1998', 'Mon DDth, YYYY'));
+
+INSERT INTO animals(species, id, name,escape_attempts, neutered, weight_kg, date_of_birth)
+VALUES ('Animal', 11, 'Ditto', 4, TRUE, 22 ,TO_DATE('May 14th, 2022', 'Mon DDth, YYYY'));
```
Comment (ShadyShawkat) [inline data.sql]: - [x] Please make sure that the entered data is the same as the data given in the exercise. Some of the given `weight_kg` values should be negatives. This is intended and not a mistake as they are going to be fixed later with queries.

File: queries.sql [hunk 3]
```diff
@@ -9,3 +9,67 @@ SELECT name FROM animals WHERE weight_kg < 10.5;
...
+SELECT neutered, MAX(escape_attempts)
+FROM animals
+WHERE escape_attempts = (
+  SELECT MAX(escape_attempts) 
+  FROM animals
+)
+GROUP BY neutered;
+
+SELECT MIN(weight_kg) FROM animals;
+SELECT MAX(weight_kg) FROM animals;
```
Comment (ShadyShawkat) [inline queries.sql]: - [x] Please make sure to select the minimum and maximum weight of each type of animal. This is just one query that selects the minimum and maximum weights for each species.

File: queries.sql [hunk 4]
```diff
@@ -9,3 +9,67 @@ SELECT name FROM animals WHERE weight_kg < 10.5;
 SELECT name FROM animals WHERE neutered = TRUE;
 SELECT name FROM animals WHERE name != 'Gabumon';
 SELECT name FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
```
Comment (ShadyShawkat) [inline queries.sql:12]: - [ ] Please make sure to write [the following query](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species%20column%20to%20unspecified.%20Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.) which updates the species in the "animals" table.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @anas599, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 4 above)
Comment (anas599) [reply queries.sql:12]: Hi @ShadyShawkat Thanks for the review,  I believe this query is from Day 1, not Day 2 which I submitted today!

**Comment by anas599** [comment]
Screenshot 2023-03-28 at 7 33 42 PM
Screenshot 2023-03-28 at 7 34 13 PM

**Review by addod19 — APPROVED** [review]
### Hi @anas599, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job addressing the comment from the previous review :rocket: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**